### PR TITLE
small clean-ups

### PR DESCRIPTION
**Pull request by pieleric**

### CODE REVIEW

File: src/odemis/driver/test/rigol_test.py [hunk 1]
```diff
@@ -75,18 +76,18 @@ def test_period(self):
...
+
         # test boundary cases
         with self.assertRaises(IndexError):
             self.wg.period.value = 0
 
+
 if __name__ == "__main__":
     unittest.main()
 
-
```
Comment (amuskens) [inline src/odemis/driver/test/rigol_test.py]: The changes look ok to me.
